Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -167,6 +166,9 @@ class MessageContentView: View {
     }
 
     func getAuthorSize(for state: State) -> CGSize {
+        // Don't show the author for the collapsed state.
+        guard state != .collapsed else { return .zero }
```

Comment:
No need to bother showing the author in the collapsed state, which further improves performance.